app_dart/lib/src/request_handling/pubsub.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:googleapis/pubsub/v1.dart';
```

Comment:
curious advantages of googleapis/pubsub over gcloud/pubsub?

Reply:
gcloud/pubsub wasn't implemented (I actually went with it first, but the prod test showed it failing)